protocol/zcash.bib

```diff
@@ -905,7 +904,7 @@ @misc{ABR1999
 
 @misc{DGKM2011,
    presort={DGKM2011},
-   author={Dana Dachman-Soled and Rosario Gennaro and Hugo Krawczyk and Tal Malkin},
+   author={Dana {Dachman-Soled} and Rosario Gennaro and Hugo Krawczyk and Tal Malkin},
```

Comment:
This should have been `{Dachman\nbd{}Soled}` but it doesn't matter in this case.